### ISSUE TITLE
Accessibility bug (Helix 2.0 beta1): Missing <label>  for <input> used for search in mobile menu

### ISSUE THREAD

**Issue by Pawel-Support** [JoomShaper/helix-ultimate#124]
Mobile Menu: There should be <label> code before the search field <input> it can be even small font - but should be.
Required by WCAG 2.1+, Level A.

How to use label but hide it: 
https://www.nomensa.com/blog/2017/how-improve-web-accessibility-hiding-elements
and
https://www.coolfields.co.uk/2011/04/accessible-forms-should-every-input-have-a-label/

**Comment by Pawel-Support** [comment]
In orginal Joomla file there is:

$output = '<label for="mod-search-searchword' . $module->id . '" class="element-invisible">' . $label . '</label> ';

**Comment by Pawel-Support** [comment]
Please for < input > add also 
` aria-label="' . $text . '"`
